README.md

```diff
@@ -43,6 +43,32 @@ export default class MyComponent extends Component {
 
 for fuller examples, see the tests directory.
 
+As with any service, if the default implementation is not suitable for testing,
+it may be swapped out during the test.
+
+```js
+import Service from '@ember/service';
+
+module('Scenario Name', function (hooks) {
```

Comment:
Not sure if it matters in this context but `hooks` isn't used here so this example is slightly misleading in that if you were to copy this exactly you wouldn't have access to `this.owner`

Reply:
that's true! at the very least we'd need a `setupTest(hooks)`

Reply:
but none of the other examples currently use `setup*` methods.

should they all be fully / working examples?